cardano-api/src/Cardano/Api/TextView.hs

```diff
@@ -73,6 +74,13 @@ data TextView = TextView
   , tvRawCBOR :: !ByteString
   } deriving (Eq, Show)
 
+instance ToJSON TextView where
+  toJSON (TextView (TextViewType tvType) (TextViewTitle title) rawCBOR) =
+    object [ "type" .= (Text.decodeUtf8 $ Base16.encode tvType)
+           , "title" .= (Text.decodeUtf8 $ Base16.encode title)
```

Comment:
```suggestion
    object [ "type" .= (Text.decodeUtf8 tvType)
           , "title" .= (Text.decodeUtf8 title)
```

Reply:
I suggest we also take the opportunity to change it to "description" rather than "title"

Reply:
How does `TextViewDescription` sound?

Reply:
For the name of the type, yeah that makes sense.